tests/installation/bootloader_uefi.pm

```diff
@@ -106,8 +106,8 @@ sub run {
         # Select boot from HDD
         send_key_until_needlematch 'inst-bootmenu-boot-harddisk', 'up';
         send_key 'ret';
-        # use firmware boot manager of aarch64 to boot HDD
-        $self->handle_uefi_boot_disk_workaround if (check_var('ARCH', 'aarch64'));
+        # use firmware boot manager of aarch64 and uefi to boot HDD
+        $self->handle_uefi_boot_disk_workaround if (check_var('ARCH', 'aarch64') || get_var('UEFI'));
```

Comment:
Yutao,

For x86_64, we should use the same workaround, right?

Reply:
I just wonder this setting will make some failure that some cases with 'UEFI' may not want to use this workaround, so are you sure such cases with UEFI could use this workaround?

Reply:
I just wonder this setting will make some failure that some cases with 'UEFI' may not want to use this workaround, so are you sure such cases with UEFI could use this workaround?

Reply:
I have verified https://openqa.suse.de/tests/6532122#

Reply:
> Yutao,
> 
> For x86_64, we should use the same workaround, right?

It won't impact x86 case.

Reply:
There are so many testcases on O3 with the bootloader_uefi.pm, so it is better to run verification logs on O3 to avoid regression.